src/browser_action/vitals.js

```diff
@@ -172,7 +178,6 @@
       console.log('[Web Vitals]', body.name, body.value.toFixed(2), body);
     }
     badgeMetrics[metricName].value = body.value;
-    badgeMetrics[metricName].final = body.isFinal;
```

Comment:
What's the context behind these `final` changes, are they related to INP?

Reply:
I dont have the full context @philipwalton may.  My understanding is that an older version of the web-vitals library used to mark a result as final, and we wouldn't badge in the extension until we knew a result was final.. but this wasn't really feasible to know and was removed from the library.

It's also a particularly poor fit as a concept for the Extension, since the whole value is to know the results as early as possible, before page unload!

Reply:
Oh, and its not INP specific.  FID was broken in the ToT Extension (and maybe the previous release?) until I changed this.

Reply:
Here's [some context around `isFinal`](https://github.com/GoogleChrome/web-vitals/issues/82). Note that it was removed from the `web-vitals` library in v1. This code may be an artifact of a prior version?

Reply:
Thanks! So this change LGTM in that case.